Meadowlark-js/backends/meadowlark-local-mongodb-opensearch/src/model/Entity.ts

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+export interface Entity {
```

Comment:
Should this be `Document` instead of `Entity`?

Reply:
Definitely 👍